src/Plugin/AsymmetricKeysGenerator/OpenSSLDefault.php

```diff
@@ -28,8 +28,7 @@ class OpenSSLDefault extends AsymmetricKeysGeneratorBase implements PluginFormIn
   public function generateAsymmetricKeys() {
     // Generate a Public/Private key pair.
     $config = array(
-      "digest_alg" => "sha512",
```

Comment:
No need for a digest_alg while generating Asymmetric keys. I used the example from PHP page, its actually incorrect there too.